src/upload-tests.proj

```diff
@@ -102,8 +102,19 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="CopyTestILCToTestHostDir" Condition="'$(BuildingUAPAOTVertical)' == 'true'">
+    <PropertyGroup>
+      <_TestILCFolderCopy>$(TestSharedFXDir)TestILC/</_TestILCFolderCopy>
```

Comment:
My latest commit addresses the renaming of these properties so they are more descriptive of what they really are